server/standard/src/main/resources/logback-debug.xml

```diff
@@ -1,29 +1,27 @@
-<configuration debug="false">
+<configuration debug="true">
```

Comment:
I'm not sure why (maybe b/c I made some minor spacing edits), but this file was a simpler rename than this diff suggests...